pages/TripDetailPage/index.js

```diff
@@ -2,22 +2,23 @@ import { useState } from "react";
 import DestinationImage from "@/components/DestinationImage";
 import TravelOverview from "@/components/TravelDetails";
 import { styled } from "styled-components";
-import Button from "@/components/Button";
+import PrimaryLink from "@/components/PrimaryButton";
```

Comment:
Please rename the file containing your link component to "PrimaryLink"

Reply:
done :)